webui/package.json

```diff
@@ -37,5 +37,8 @@
     "vite-plugin-top-level-await": "^1.1.1",
     "vite-plugin-wasm": "^2.1.0"
   },
-  "type": "module"
+  "type": "module",
```

Comment:
I am not a nodejs expert but I think that webui is not a dependency of the webui package itself

Reply:
sorry this shouldn't be in PR